identity/src/main/java/com/android/identity/ProgressListener.java

```diff
@@ -0,0 +1,5 @@
+package com.android.identity;
+
+public interface ProgressListener {
+  void onProgressUpdate(float percentComplete);
```

Comment:
I'd rather see the progress expressed as integers. Something like `onProgressUpdate(int progress, int max)`. If someone wants to do the arithmetic to get a percentage, easy peasy.

This has the bonus of avoiding any float shenanigans (which I think you're mitigating elsewhere with a `Min(1f, percent)` operation.

@davidz25 WDYT?

Reply:
Done